pkgs/development/compilers/openjdk/21.nix

```
@@ -29,7 +29,7 @@ let
       owner = "openjdk";
       repo = "jdk${version.feature}u";
       rev = "jdk-${version.feature}${version.interim}+${version.build}";
-      hash = "sha256-fA8nRWBuTL87S8mwapmNfCPPQoI2aKHjbHJ6PDN3khs=";
+      hash = "sha256-Qoaf4drhhhy11SD+3FBcKQ1jWSlwODD76z+wYONBHfo=";
```

```suggestion
      hash = "sha256-zRN16lrc5gtDlTVIQJRRx103w/VbRkatCLeEc9AXWPE=";
```
Hash changed upstream ?

Also commit title should be 

openjdk: 21+35 -> 21.0.3+9

---

Good catch!